it/test/uk/gov/hmrc/tradergoodsprofilesrouter/RequestAdviceIntegrationSpec.scala

```diff
@@ -33,25 +33,24 @@ class RequestAdviceIntegrationSpec
     with GetRecordsDataSupport
     with BeforeAndAfterEach {
 
-  private val correlationId                      = "d677693e-9981-4ee3-8574-654981ebe606"
-  private val timestamp                          = "2021-12-17T09:30:47Z"
-  private val eori                               = "GB123456789001"
-  private val recordId                           = "8ebb6b04-6ab0-4fe2-ad62-e6389a8a204f"
-  private val url                                = fullUrl(s"/traders/$eori/records/$recordId/advice")
+  private val correlationId              = "d677693e-9981-4ee3-8574-654981ebe606"
+  private val timestamp                  = "2021-12-17T09:30:47Z"
+  private val eori                       = "GB123456789001"
+  private val recordId                   = "8ebb6b04-6ab0-4fe2-ad62-e6389a8a204f"
+  private val url                        = fullUrl(s"/traders/$eori/records/$recordId/advice")
   override def hawkConnectorPath: String = "/tgp/getrecords/v1"
-  def pegaConnectorPath: String = "/tgp/createaccreditation/v1"
+  def pegaConnectorPath: String          = "/tgp/createaccreditation/v1"
 
-  override def extraApplicationConfig: Map[String, Any] = {
+  override def extraApplicationConfig: Map[String, Any] =
     Map(
       s"microservice.services.hawk.host" -> wireMockHost,
```

Comment:
can we remove the s before "", please? We do not need these.

Reply:
Done